controllers/controllers/networking/cfroute_controller_test.go

```diff
@@ -151,6 +152,10 @@ var _ = Describe("CFRouteReconciler Integration Tests", func() {
 		}).Should(Succeed())
 	})
 
+	It("writes a log message", func() {
```

Comment:
Again, this logger is passed directly from `shared.ObjectLogger()` to `createValidRouteStatus()`, so the logger can never be a discard logger and it's not worth testing.

Rather check the loggers used in the finalize and createOrPatch* methods.